tests/Unit/Domain/CoordinateMaps/TestMapHelpers.hpp

```diff
@@ -338,6 +338,71 @@ void test_suite_for_map_on_unit_cube(const Map& map) {
   test_helper(map2);
 }
 
+/*!
+ * \ingroup TestingFrameworkGroup
+ * \brief Given a Map `map`, tests the map functions, including map inverse,
```

Comment:
document or rename `max_radius` such that it is clear that it the radius used for the boundary of the map and that the test expects that the map still be valid on the boundary.

You also might want to differentiate the documentation for this test from that for test_suite_for_map_on_unit_cube, also because this test only works for 3D and only the code mentions that